admin-dev/themes/default/template/controllers/orders/form.tpl

```diff
@@ -351,7 +351,7 @@
 				id_product: id_product,
 				id_product_attribute: id_product_attribute,
 				id_customer: id_customer,
-				price: new Number(new_price.replace(",",".")).toFixed(4).toString()
+				price: ps_round(new Number(new_price.replace(",",".")), 6).toString(),
```

Comment:
I don't know,  the feature has been tested both on chrome and firefox browser, in ubuntu and Windows 7 operating systems.